### ISSUE TITLE
Web Login: Incorrect password message is misleading 

### ISSUE THREAD

**Issue by kazemisoroush** [MadAppGang/identifo#222]
As a user, I want to see a message like "Invalid credentials" when I enter the wrong password. Instead, I see "User not found" which is misleading.

![image](https://user-images.githubusercontent.com/8931595/133718169-a3d642d2-e680-42e4-85f8-f110ad29d99c.png)

Here is the error message for the same request:
```
API_ROUTER: 2021/09/17 03:08:47 router.go:152: api error: error.api.request.incorrect_login_or_password (status=401). Details: User not found. . Where: LoginWithPassword.CheckPassword.
[negroni] 2021-09-17T03:08:47Z | 401 |   106.544718ms | identifo.dev.evergen.technology | POST /auth/login
A
```

**Comment by kazemisoroush** [comment]
This is happening in v2.1.2

**Comment by erudenko** [comment]
The reason for this message is security.

knowing the user name or email you can guess if the user registered in the system. This information is private information itself. That is why when the credentials are wrong we should tell something `user is not found or credentials are wrong`

**Comment by kazemisoroush** [comment]
I agree with preventing the security issue but the `Invalid credentials` error message won't give any information about the user exists or not.
